public/js/viz-scattercluster.js

```diff
@@ -0,0 +1,115 @@
+// TODO rewrite scatterPlot and controls as an object
+var renderScatterCluster = function(sampleData, k) {
+
+    // delete any scatterplots already plotted
+    d3.select('#scattercluster > svg').remove();
```

Comment:
Haha yeah this is crazy! =D Can't one recolour existing dots? If so but you don't want to mess with it in this PR, please add an issue and respond here so I know it's "resolved". =) But we are accumulating a worrying amount of technical debt...